code/modules/reagents/chemistry/reagents/toxin_reagents.dm

```diff
@@ -490,6 +490,15 @@
 	toxpwr = 0.5
 	taste_description = "bad cooking"
 
+//MonkeStation Edit: Bad Food can cause disease
+/datum/reagent/toxin/bad_food/reaction_mob(mob/living/M, method = INGEST, reac_volume, show_message, touch_protection)
+	. = ..()
+	if((ishuman(M) || ismonkey(M)) && prob(20))
```

Comment:
```suggestion
	if(prob(20) && (ishuman(M) || ismonkey(M))
```
super minor thing, theres just no point checking if the mob is human/monkey if the prob check fails